data/human/deep missions.txt

```diff
@@ -2735,8 +2735,8 @@ mission "Stone of our Fathers 5"
 			`	"The day I struck the mother lode on Helheim."`
 			`	"The day I met <first>, who helped me find my family again."`
 			`	"The day I saw my son again, even though he would not see me."`
-			`	Thorleif sniffs, takes a moment to write something down he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
-			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that, I thank you, father."`
+			`	Thorleif sniffs, takes a moment to write down something he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
+			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that I thank you, father."`
```

Comment:
I'm not sure about the changes here, honestly, but the rest of it seems good to me!

Reply:
I changed this specifically because it is taking the character's particular way of speaking and bleeds in into the voice of the narrator.

Reply:
In that case, that makes sense.